src/nvim/buffer.c

```diff
@@ -3570,20 +3570,6 @@ void fname_expand(buf_T *buf, char_u **ffname, char_u **sfname)
   if (*sfname == NULL)          /* if no short file name given, use ffname */
     *sfname = *ffname;
   *ffname = (char_u *)fix_fname((char *)*ffname);     /* expand to full path */
-
-#ifdef FEAT_SHORTCUT
```

Comment:
`FEAT_SHORTCUT` seems to be used for Windows in legacy Vim. Can we be sure that we won't need this feature?